### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by anugs**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @anugs, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16829).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/installation/index.html [hunk 1]
```diff
@@ -9,29 +9,29 @@
...
-    <p>In addition there are two installation methods available for experienced users:</p>
+    <p>There are two alternative installation methods available for experienced users:</p>
     <ul>
         <li><b>Home Assistant Supervised</b>: Manual installation of the Supervisor.</li>
-        <li><b>Home Assistant Core</b>: Manual installation using a Python virtual environment.</li>
+        <li><b>Home Assistant Core</b>: Manual installation using Python virtual environment.</li>
     </ul>
     <p>
-        The list below shows which installation method is available on what device or platform.
+        The list below shows the compatibility of the above installation methods with the different devices and platforms.
```
Comment (frenck) [inline source/installation/index.html]: It isn't aimed at compatibility, I think that is misleading in this new text.

Comment (anugs) [reply source/installation/index.html]: 'The list below shows the compatibility of the above installation methods with the different device or platform' can be modified as 'The list below shows the installation method available based on the device and platform being used'. Please let me know if this is better.

**Review by frenck — APPROVED** [review]
Thanks, @anugs 👍

**Pull request merged**